contracts/upgradeability/UpgradeabilityProxy.sol

```diff
@@ -17,13 +17,19 @@ contract UpgradeabilityProxy is Proxy {
    */
   event Upgraded(address implementation);
 
-  // Storage slot of the address of the current implementation
-  bytes32 private constant IMPLEMENTATION_SLOT = keccak256("org.zeppelinos.proxy.implementation");
+  /**
+   * @dev Storage slot with the address of the current implementation.
+   * @dev This is the hash of "org.zeppelinos.proxy.implementation", and is
+   * @dev validated in the constructor.
+   */
+  bytes32 private constant IMPLEMENTATION_SLOT = 0x7050c9e0f4ca769c69bd3a8ef740bc37934f8e2c036e5a723fd8ee048ed3f8c3;
```

Comment:
Why this change was made?

Reply:
Because Solidity inlined a `keccak256` calculation everywhere the variable was used, instead of calculating the value during compilation. This resulted in a lot of additional cost for every interaction with the proxy.